lib/AppConfig.php

```diff
@@ -86,7 +89,7 @@ public function setUserValue($userId, $key, $value) {
 	 * @return bool
 	 */
 	public function enterpriseFeaturesEnabled() {
-		if (!$this->appManager->isEnabledForUser('enterprise_key') && !\getenv('CI')) {
+		if (!$this->licenseManager->checkLicenseFor($this->appName)) {
```

Comment:
`&& !\getenv('CI')` still required

Reply:
Not anymore. We have the grace period now which works also in CI

Reply:
For reference: https://github.com/owncloud/core/blob/master/tests/acceptance/run.sh#L774
Starts tests with the grace period popup disabled.